nh_clinical/tests/__init__.py

```diff
@@ -2,27 +2,22 @@
 # -*- coding: utf-8 -*-
 
 # Base level tests
-from . import test_location
-from . import test_operations
-from . import test_user_management
-from . import test_api_demo
 from . import test_activity_extension
+from . import test_api_demo
 from . import test_base_extensions
+from . import test_location
+from . import test_operations
 from . import test_patient_placement_wizard
 from . import test_responsibility_allocation_wizard
+from . import test_user_management
 from . import test_users
 
 # Test Patient
 from . import test_patient
```

Comment:
Hehe, I like that you're pushing for the refactors so much, should I move them into a sub-package called `patient` or `nh_clinical_patient` though? It depends if we want our sub-packages 1:1 with the python module or 1:1 with the Odoo model.

Advantages of sub-packages mirroring the python module is that I think it's a little more straight forward to find the tests relating to the file you are currently looking at. I also think that in the minority of cases where we will choose to house multiple classes in a single python module there is probably a good reason why we want to group them together and so we may also want the tests to be grouped together.

Advantages of the sub-packages mirroring individual classes is that the test file naming can be shorter because there's no need to distinguish between multiple classes.